plugins/rest/auth.go

```diff
@@ -145,22 +145,41 @@ func (ap *bearerAuthPlugin) NewClient(c Config) (*http.Client, error) {
 }
 
 func (ap *bearerAuthPlugin) Prepare(req *http.Request) error {
+	token, err := ap.buildToken()
+	if err != nil {
+		return err
+	}
+
+	req.Header.Add("Authorization", fmt.Sprintf("%v %v", ap.Scheme, token))
+	return nil
+}
+
+func (ap *bearerAuthPlugin) AuthHeader() (http.Header, error) {
+	token, err := ap.buildToken()
+	if err != nil {
+		return nil, err
+	}
+
+	header := make(http.Header)
+	header.Add("Authorization", fmt.Sprintf("%v %v", ap.Scheme, token))
+	return header, err
```

Comment:
```suggestion
	return header, nil
```
just a little bit more explicit 🤷

Reply:
done